src/F2_autoreport.py

```diff
@@ -23,6 +47,9 @@
     'font-inpbox': '#FFFFFF'
 }
 # Reading the relevant user-inputs from the csv files into Pandas dataframes
+
+
+
 dfprojectData = pd.read_csv('../inputs/csv_elements/project_data.csv')
```

Comment:
Did you run `black .` prior to committing? I am not sure why you commit 3 empty spaces?

Reply:
Oops. This is a mistake. I wanted to read the data just below the three empty lines from JSONs as well. But then postponed it and forgot to remove these lines. Consider them placeholders xD